core-services/chat/src/main/java/org/silverpeas/core/chat/ChatUser.java

```diff
@@ -0,0 +1,279 @@
+package org.silverpeas.core.chat;
+
+import org.silverpeas.core.admin.user.constant.UserAccessLevel;
+import org.silverpeas.core.admin.user.constant.UserState;
+import org.silverpeas.core.admin.user.model.User;
+import org.silverpeas.core.admin.user.model.UserDetail;
+import org.silverpeas.core.chat.servers.ChatServer;
+import org.silverpeas.core.personalization.UserPreferences;
+import org.silverpeas.core.util.SettingBundle;
+import org.silverpeas.core.util.SilverpeasBundleList;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.silverpeas.core.util.SilverpeasBundleList'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)